index.js

```diff
@@ -112,16 +114,16 @@ NanoEvents.prototype = {
    */
   emit: function emit (event) {
     var list = this.events[event]
-    if (!list || !list.length) return false
+    if (!list || !list[0]) return false
 
-    var copy = list.slice(0)
+    list = list.slice()
 
-    var args = [].slice.call(arguments, 1)
-
-    for (var i = 0; i < copy.length; i++) {
-      var l = copy[i]
+    for (
```

Comment:
This trick is too tricky :D

Reply:
Feel free to cherry pick changes you like and drop the ones you don't like

Reply:
Sure! I will revert it.